js/demo-app/src/index.js

```diff
@@ -201,18 +211,21 @@ function updateActiveNav(navConfigs = NAV_CONFIGS, fqRoute = window.location.has
       let activeNavItem = navConfigs.find(toMatch => {
         return (toMatch.assignedRoute === currRoute);
       });
-
       activeNavId = (activeNavItem ? activeNavItem.id : null);
     }
+  } else {
```

Comment:
Here's where I was talking.  If you pull out this conditionally and just search the fqRoute, you should be able to use it for programmic or url versions.